pkg/apis/ocs/v1/storagecluster_types.go

```diff
@@ -2,6 +2,7 @@ package v1
 
 import (
 	conditionsv1 "github.com/openshift/custom-resource-status/conditions/v1"
+	rookv1 "github.com/rook/rook/pkg/apis/rook.io/v1"
 	rookalpha "github.com/rook/rook/pkg/apis/rook.io/v1alpha2"
```

Comment:
If we have `v1`, why do we still need `v1alpha2`?
We should remove it.

Reply:
Fixed that